.github/dependabot.yml

```diff
@@ -4,6 +4,7 @@ updates:
     directory: /
     schedule:
       interval: daily
+    target-branch: develop
     allow:
```

Comment:
This is a strange configuration... We should probably drop this entire hash, honestly.